gitlab/v4/objects/__init__.py

```diff
@@ -15,63 +15,63 @@
-from .push_rules import *
-from .releases import *
-from .runners import *
-from .services import *
-from .settings import *
-from .sidekiq import *
-from .snippets import *
-from .statistics import *
-from .tags import *
-from .templates import *
-from .todos import *
-from .triggers import *
-from .users import *
-from .variables import *
-from .wikis import *
-from .personal_access_tokens import *
+from .access_requests import *  # noqa: F401,F403
+from .appearance import *  # noqa: F401,F403
+from .applications import *  # noqa: F401,F403
+from .audit_events import *  # noqa: F401,F403
+from .award_emojis import *  # noqa: F401,F403
+from .badges import *  # noqa: F401,F403
+from .boards import *  # noqa: F401,F403
+from .branches import *  # noqa: F401,F403
+from .broadcast_messages import *  # noqa: F401,F403
+from .clusters import *  # noqa: F401,F403
+from .commits import *  # noqa: F401,F403
+from .container_registry import *  # noqa: F401,F403
+from .custom_attributes import *  # noqa: F401,F403
+from .deploy_keys import *  # noqa: F401,F403
+from .deployments import *  # noqa: F401,F403
+from .deploy_tokens import *  # noqa: F401,F403
+from .discussions import *  # noqa: F401,F403
+from .environments import *  # noqa: F401,F403
+from .epics import *  # noqa: F401,F403
+from .events import *  # noqa: F401,F403
+from .export_import import *  # noqa: F401,F403
+from .features import *  # noqa: F401,F403
+from .files import *  # noqa: F401,F403
+from .geo_nodes import *  # noqa: F401,F403
+from .groups import *  # noqa: F401,F403
+from .hooks import *  # noqa: F401,F403
+from .issues import *  # noqa: F401,F403
+from .jobs import *  # noqa: F401,F403
+from .labels import *  # noqa: F401,F403
+from .ldap import *  # noqa: F401,F403
+from .members import *  # noqa: F401,F403
+from .merge_request_approvals import *  # noqa: F401,F403
+from .merge_requests import *  # noqa: F401,F403
+from .milestones import *  # noqa: F401,F403
+from .namespaces import *  # noqa: F401,F403
+from .notes import *  # noqa: F401,F403
+from .notification_settings import *  # noqa: F401,F403
+from .packages import *  # noqa: F401,F403
+from .pages import *  # noqa: F401,F403
+from .pipelines import *  # noqa: F401,F403
+from .projects import *  # noqa: F401,F403
+from .push_rules import *  # noqa: F401,F403
+from .releases import *  # noqa: F401,F403
+from .runners import *  # noqa: F401,F403
+from .services import *  # noqa: F401,F403
+from .settings import *  # noqa: F401,F403
+from .sidekiq import *  # noqa: F401,F403
+from .snippets import *  # noqa: F401,F403
+from .statistics import *  # noqa: F401,F403
+from .tags import *  # noqa: F401,F403
+from .templates import *  # noqa: F401,F403
+from .todos import *  # noqa: F401,F403
+from .triggers import *  # noqa: F401,F403
+from .users import *  # noqa: F401,F403
+from .variables import *  # noqa: F401,F403
+from .wikis import *  # noqa: F401,F403
+from .personal_access_tokens import *  # noqa: F401,F403
```

Comment:
Thank you! :pray: I think I wanted to do something with `__all__` to prevent this but it was too messy. :grin: 

I wonder if a per-file ignore would be convenient in this specific case: https://stackoverflow.com/a/55746853

Reply:
Done.